types/carbon__motion/index.d.ts

```diff
@@ -0,0 +1,25 @@
+// Type definitions for carbon__motion-ts 0.0
+// Project: https://github.com/baz/foo (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
```

Comment:
I noticed you fixed the headers on PR #32862. I'm  not sure if my merging this one it will override your fixes. I don't think it's going to happen but I would recommend to check, just in case.

Reply:
Ah yeah looks like I forked off of that branch rather than master. I was able to download the types from both carbon__type and carbon__motion without issue so I think you were right.